sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -86,7 +87,7 @@ export interface AssessmentConsolidatedData {
   studentDataPartnerStudyWeeks?: number;
   studentDataPartnerEmploymentInsurance?: YesNoOptions;
   studentDataPartnerFedralProvincialPDReceiptCost?: number;
-  studentDataParentDependentTable?: unknown;
+  studentDataParentDependentTable?: JSONDoc;
```

Comment:
just to check, should this be a list of JSONDoc

Reply:
May be. But we never populated data to these values in tests, so kept it as JSONDoc. When we create an interface to this value in future we can update to list of that interface that extends JSONDoc.